web_src/js/features/repo-issue-edit.js

```diff
@@ -121,14 +122,22 @@ async function onEditContent(event) {
     hideElem(editContentZone);
     const dropzoneInst = comboMarkdownEditor.attachedDropzoneInst;
     try {
+      const version = editContentZone.getAttribute('data-version');
+
       const params = new URLSearchParams({
         content: comboMarkdownEditor.value(),
         context: editContentZone.getAttribute('data-context'),
+        version,
       });
       for (const fileInput of dropzoneInst?.element.querySelectorAll('.files [name=files]')) params.append('files[]', fileInput.value);
 
       const response = await POST(editContentZone.getAttribute('data-update-url'), {data: params});
       const data = await response.json();
+      if (response.status === 400) {
+        showErrorToast(data.errorMessage);
+        return;
+      }
+      editContentZone.setAttribute('data-version', parseInt(version) + 1);
```

Comment:
Maybe we should get the version from the response?

Reply:
should refetch issue after update or just increment and send as response?

Reply:
I would like the latter.

Reply:
done